Базовая стилизация всех страниц

R-Roman-R opened the pull request:
Подключил стили к страницам.
Добавил фоновые изображения и декоративные элементы.
Добавил тексту параметры из макета. Добавил div-обертку с классом "inner-background" в catalog для разметки фона. Добавил классы некоторым тегам для более удобной стилизации. Поправил вложенность тегов. Исправил грамматические ошибки.

---
:mortar_board: [Базовая стилизация всех страниц](https://up.htmlacademy.ru/htmlcss/33/user/1852641/tasks/6)

Commit:
Создал папку и файл со стилями

Commit:
Добавил базовую стилизацию всех страниц.
Добавил фоновые изображения (в т.ч. декоративные).
Добавил тексту параметры из макета.

senchkim commented on catalog.html:
```diff
@@ -112,107 +113,109 @@ <h2 class="visually-hidden">Поиск по фильтрам</h2>
-            <span class="hotel-card-rating">Рейтинг: 9,2</span>
-          </li>
-        </ul>
+          <ul class="hotel-list">
```
Огромное спасибо за табуляцию 👍 
Аж читать приятно)

senchkim commented on styles/styles.css:
```diff
@@ -0,0 +1,429 @@
+}
+
+body {
+  margin: 0;
```
Еще бы неплохо сюда добавить `padding: 0;`

senchkim commented on styles/styles.css:
```diff
@@ -0,0 +1,429 @@
+}
+
+.navigation-item {
+  list-style-type: none;
```
- Для обнуления списка нужно добавить еще 2 свойства:
1) `padding: 0;`
2) `margin: 0;`
- Здесь и ниже ты по сути, дублируешь обнуление свойств списка - вопрос зачем?
1) Можно же просто через запятую перечислить классы у списка через запятую и задать им ОДИН раз нужные свойства 👍

senchkim commented on styles/styles.css:
```diff
@@ -0,0 +1,429 @@
+  text-transform: uppercase;
+}
+
+.search p {
```
Плохо стилизовать по тэгу(
Лучше заменить на класс так как стили не должны знать ничего про тэги - а вдруг тэг кто-то случайно поменяет тэг и что тогда? Все стили поломаются(

senchkim commented on styles/styles.css:
```diff
@@ -0,0 +1,429 @@
+  text-transform: uppercase;
+}
+
+.newsletter p {
```
Здесь тоже тэг лучше заменить на класс 👍

senchkim commented on styles/styles.css:
```diff
@@ -0,0 +1,429 @@
+  color: #000000;
+  background-color: #ffffff;
+  text-align: center;
+}
```
https://codeguide.academy/html-css.html#css-import
Советую прочесть и по возможности придерживаться код-гайда академии - это сделает тебя сильнее как разработчика 💯

senchkim commented:
В целом отличная работа хвалю 👍

Commit:
Доработал стили.
Обнулил списки, сгруппировал их в одно правило.
Удалил дублирующиеся свойства.
Добавил классы параграфам, исправил мелкие ошибки.
